roles/openshift_logging/defaults/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+openshift_logging_curator_image_prefix: "{{ openshift_logging_image_prefix | default(__openshift_logging_image_prefix) }}"
```

Comment:
it appears to be set correctly to `docker.io/openshift/` when testing the `origin` deployment type and setting neither `openshift_logging_image_prefix`, nor `openshift_logging_*_image_prefix` vars. 

What leads you to think that `__openshift_logging_image_prefix` won't be set correctly? Is it [variable precedenece](http://docs.ansible.com/ansible/latest/playbooks_variables.html), because should apply in our favour in this scenario. Or is it something else I may be missing?